gui/test/setup/renderer.ts

```diff
@@ -1,9 +1,14 @@
+import log from 'electron-log';
 import Enzyme from 'enzyme';
 import ReactSixteenAdapter from 'enzyme-adapter-react-16';
 import chai from 'chai';
 import spies from 'chai-spies';
 import chaiAsPromised from 'chai-as-promised';
 
+log.transports.console.level = false;
+log.transports.file.level = false;
+log.transports.mainConsole = null;
```

Comment:
FYI this is the IPC transport that broadcasts logs from renderer -> main console. The `console` transport level does not affect it, so we have to disable it to avoid the logs from appearing in tests.

Related: https://github.com/megahertz/electron-log/issues/138